bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/BugsnagExitInfoPlugin.kt

```diff
@@ -23,9 +23,11 @@ class BugsnagExitInfoPlugin @JvmOverloads constructor(
             )
         }
 
-        val exitInfoPluginStore = ExitInfoPluginStore(client.immutableConfig)
-        val oldPid = exitInfoPluginStore.load()
-        exitInfoPluginStore.persist(android.os.Process.myPid())
+        val exitInfoPluginStore =
+            ExitInfoPluginStore(client.immutableConfig)
+        val oldPid = exitInfoPluginStore.load().first
+        val ExitInfoKeys = exitInfoPluginStore.load().second
```

Comment:
calling `load()` twice is very expensive:

```suggestion
        val (oldPid, exitInfoKeys) = exitInfoPluginStore.load()
```